content/resources/user/submit-data/sower.md

```diff
@@ -0,0 +1,24 @@
+---
+title: "Gen3 - Data Contribution"
+date: 2018-09-12T20:53:20-05:00
+linktitle: /resources/user/submit-data
```

Comment:
should this be "/resources/user/submit-data/sower" ? 
in alpha, when I follow this link, I get an error:
https://gen3.org/resources/user/submit-data/sower